### PR TITLE
Add FastAPI interface between the game and map.

### PR DESCRIPTION
**Pull request by DanAlbert**
A possible explanation for the infrequent CTDs we've been seeing since
adding fast forward is that QWebChannel doesn't keep a reference to the
python objects that it passes to js, so if the object is GC'd before the
front end is done with it, it crashes.

We don't really like QWebChannel anyway, so this begins replacing that
with FastAPI.

### CODE REVIEW

File: game/server/debuggeometries/routes.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+def find_flight(game: Game, flight_id: UUID) -> Flight:
+    for coalition in game.coalitions:
+        for package in coalition.ato.packages:
+            for flight in package.flights:
+                if flight.id == flight_id:
+                    return flight
+    raise KeyError(f"No flight found with ID {flight_id}")
+
+
+@router.get("/hold/{flight_id}")
```
Comment (SnappyComebacks) [inline game/server/debuggeometries/routes.py:23]: Do you think these might expand to `router.get("/something/{package_id}"` at some point?  I get why you are doing `flight_id`, just wondered if we are going to shoot ourselves in the foot later on.  Or YAGNI I guess.

File: game/config.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # Set to True to enable computing expensive debugging information. At the time of
 # writing this only controls computing the waypoint placement zones.
 ENABLE_EXPENSIVE_DEBUG_TOOLS = False
+
+SERVER_BIND_ADDRESS = "127.0.0.1"
+SERVER_PORT = 5000
```
Comment (SnappyComebacks) [inline game/config.py]: I didn't know the current system at all, will we have to make this a configurable item at some point?  I could see some long term use to doing so for the dedicated server crowd.

File: game/server/debuggeometries/routes.py (hunk 1 above)
Comment (DanAlbert) [reply game/server/debuggeometries/routes.py:23]: Yeah, eventually there will be endpoints for all the things. The flight is just the thing I needed for this.

File: game/config.py (hunk 2 above)
Comment (DanAlbert) [reply game/config.py]: Yeah, definitely eventually. Right now it'd be worthless, and even if the UI could run in a browser, we'd need to work out multi-user crap...

**Pull request merged**